content/posts/customizing/dark-theme/index.fr.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Activer le mode sombre"
+date: 2022-06-12T01:30:50+06:00
+author:
+  name: Emruz Hossain
```

Comment:
Please, use your name as author. Add your image into `images/author` directory.

Reply:
Hi @hossainemruz,

Oooh, OK as you want :). I see that next time next week.

Reply:
Hi @hossainemruz, it's done. I think you can finaly merge now when you're ready.